README.md

```diff
@@ -94,6 +94,21 @@ Duplicate `user` entries are not accepted, and only the first instance of that `
 **Usage**:</br>
 &ensp;`python get_backend_info.py <hub> <-group> <-project> --full_data`
 
+-----
+
+### edit_backends.py
+**What it does**: Adds or Removes a backend from a project within your Hub</br>
+**Parameters**:</br>
+&ensp;`<hub>`: Required. The name of your hub.</br>
+&ensp;`<action>`: Required. Either `add` or `remove`.</br>
+&ensp;`<group>`: Required. The name of the parent group.</br>
+&ensp;`<project>`: Required. The name of the parent project.</br>
+&ensp;`<backend>`: Required. The name of the backend to add or remove.</br>
+&ensp;`<-priority>`: Required for `add` action. Integer priority of the backend, between 1 and 10,000.</br>
+
+**Usage**:</br>
+&ensp;`python edit_backends.py <hub> <action> <group> <project> <backend_name> <-priority>`
```

Comment:
Would it be possible to have the usage be more obvious for users unaware with inputting parameters in the terminal that `-priority` would be a flag that required the actual priority input afterwards. For example, some users may think that a valid run command would be 
`python edit_backends.py some-hub add some-group some-project backend_name 1`, 
which would fail saying there were unrecognized arguments. I think to remedy this we could change the Usage line to say:
`python edit_backends.py <hub> <action> <group> <project> <backend_name> -priority <priority>`
or something similar to that. That way the user knows they must put the priority flag and a priority entry afterwards.

Could also add a Usage line specific for adding and removing a backend, since they differ. An example of this in the [edit_group.py usage docs](https://github.com/ibm-quantum/Hub-Automation#edit_grouppy)

Reply:
I made the suggested changes to the readme